src/modules/IscsiClientLib.rb

```diff
@@ -473,12 +473,20 @@ def ScanDiscovered(data)
       target = ""
       portal = ""
       iface = ""
+      dumped = true
       Builtins.y2milestone("Got data: %1", data)
 
       Builtins.foreach(data) do |row|
         row = Builtins.substring(row, Builtins.findfirstnotof(row, "\t "), 999)
         if Builtins.search(row, "Target:") != nil
+          if !dumped
+            # don't add Scope:Link IPv6 address
+            if !portal.start_with?("[fe80:")
+              ret = ret << "#{portal} #{target} #{iface}"
```

Comment:
this line is strange as operator << works inplace so it should be `ret << "#{portal} #{target} #{iface}"` only.
Also please document what means that variable dumped. What is logic behind. Code was not perfectly understandable and new variable for special dumping does not help much. So why it is added only when it found another Target? It is because Target is start of entry?